app/src/main/java/com/jerboa/ui/components/common/MarkdownHelper.kt

```diff
@@ -165,28 +186,32 @@ object MarkdownHelper {
         color: Color = Color.Unspecified,
         onClick: (() -> Unit)? = null,
         onLongClick: (() -> Unit)? = null,
+        maxLines: Int? = null,
+        style: TextStyle? = null,
+        removeImages: Boolean = false,
```

Comment:
As a convention, I think having a negative parameter like this is a bit confusing, I'd rather do `enableImages`

Reply:
updated